tests/fixtures.py

```diff
@@ -36,48 +37,63 @@
 
 
 JSON_WORKFLOW_TASKS = {
+    "error": "None",
     "data": [
         {
-            "id": "c0d04ec3-98d7-4183-902f-5bcb2a176d89",
+            "id": "aa2cba17-d35c-4395-ab01-a0fd8191a4b3",
             "name": "sobloo-s2-l1c-aoiclipped:1",
-            "blockVersionTag": "2.2.2",
+            "parentsIds": [],
+            "blockName": "sobloo-s2-l1c-aoiclipped",
+            "blockVersionTag": "2.3.0",
             "block": {
+                "id": "3a381e6b-acb7-4cec-ae65-50798ce80e64",
                 "name": "sobloo-s2-l1c-aoiclipped",
+                "displayName": "Sentinel-2 L1C MSI AOI clipped",
                 "parameters": {
-                    "nodata": {
-                        "type": "number",
-                    },
+                    "ids": {"type": "array", "default": "None"},
+                    "bbox": {"type": "array", "default": "None"},
                     "time": {
                         "type": "dateRange",
                         "default": "2018-01-01T00:00:00+00:00/2020-12-31T23:59:59+00:00",
                     },
                 },
+                "type": "DATA",
+                "isDryRunSupported": "true",
+                "version": "2.3.0",
             },
+            "environment": "None",
         },
         {
-            "id": "af626c54-156e-4f13-a743-55efd27de533",
+            "id": "24375b2a-288b-46c8-b404-53e48d4e7b25",
             "name": "tiling:1",
-            "blockVersionTag": "1.0.0",
+            "parentsIds": ["aa2cba17-d35c-4395-ab01-a0fd8191a4b3"],
+            "blockName": "tiling",
+            "blockVersionTag": "2.2.3",
             "block": {
+                "id": "3e146dd6-2b67-4d6e-a422-bb3d973e32ff",
                 "name": "tiling",
+                "displayName": "Raster Tiling",
                 "parameters": {
                     "nodata": {
                         "type": "number",
-                        "default": None,
-                        "required": False,
-                        "description": "Value representing..",
+                        "default": "None",
+                        "required": "false",
+                        "description": "Value representing ...",
                     },
                     "tile_width": {
                         "type": "number",
                         "default": 768,
-                        "required": True,
+                        "required": "true",
```

Comment:
Why does this need to change? In general I think when you define a dictionary, it should have boolean values set to `True` and `False` (they get converted from json `true` and `false) and the same with `None` (gets converted from json `null`).

Reply:
oh right, fixed it